response_operations_ui/views/messages.py

```diff
@@ -183,6 +199,11 @@ def _get_conversation_breadcrumbs(messages):
     ]
 
 
+def get_pagination(**kwargs):
+    kwargs.setdefault('record_name', 'records')
+    return Pagination(show_single_page=False, **kwargs)
```

Comment:
Yeah, I forgot to change this, my bad, this could either all be in the view or left as `get_pagination` which do you think is the most readable/nicest?

Reply:
I'd say all in the view and remove the method as it's only used in one place.